Drop-In/src/test/java/com/braintreepayments/api/BaseActivityUnitTest.java

```diff
@@ -1,106 +0,0 @@
-package com.braintreepayments.api;
-
-import android.content.Intent;
-
-import org.json.JSONException;
-import org.json.JSONObject;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.robolectric.Robolectric;
-import org.robolectric.RobolectricTestRunner;
-import org.robolectric.android.controller.ActivityController;
-import org.robolectric.shadows.ShadowActivity;
-
-import static androidx.appcompat.app.AppCompatActivity.RESULT_FIRST_USER;
-import static androidx.appcompat.app.AppCompatActivity.RESULT_OK;
-import static junit.framework.Assert.assertEquals;
-import static junit.framework.Assert.assertFalse;
-import static junit.framework.Assert.assertNotNull;
-import static junit.framework.Assert.assertNull;
-import static junit.framework.Assert.assertTrue;
-import static org.robolectric.Shadows.shadowOf;
-
-@RunWith(RobolectricTestRunner.class)
-public class BaseActivityUnitTest {
-
-    private ActivityController mActivityController;
-    private BaseActivity mActivity;
-
-    @Test
-    public void onCreate_setsDropInRequest() {
-        Intent intent = new Intent();
-        intent.putExtra(DropInClient.EXTRA_CHECKOUT_REQUEST, new DropInRequest());
-        mActivityController = Robolectric.buildActivity(BaseActivity.class, intent);
-        mActivity = (BaseActivity) mActivityController.get();
-
-        assertNull(mActivity.mDropInRequest);
-
-        mActivityController.create();
-
-        assertNotNull(mActivity.mDropInRequest);
-    }
-
-    @Test
-    public void getDropInClient_returnsADropInClient() {
-        Intent intent = new Intent();
-        intent.putExtra(DropInClient.EXTRA_CHECKOUT_REQUEST, new DropInRequest());
-        setup(intent);
-
-        assertNotNull(mActivity.getDropInClient());
-    }
-
-    @Test
-    public void getDropInClient_setsClientTokenPresentWhenAClientTokenIsNotPresent() {
-        Intent intent = new Intent();
-        intent.putExtra(DropInClient.EXTRA_CHECKOUT_REQUEST, new DropInRequest());
-        setup(intent);
-
-        mActivity.getDropInClient();
-
-        assertFalse(mActivity.mClientTokenPresent);
-    }
```

Comment:
Should these be tested for DropInActivity now too?

Reply:
When I was removing `BaseActivity` I couldn't see the value in these tests. We should have enough coverage in `DropInActivityUnitTest`.